src/bootman/kernel.c

```diff
@@ -569,6 +572,11 @@ bool boot_manager_install_kernel_internal(const BootManager *manager, const Kern
                 return false;
         }
 
+        /* Don't rewrite the same initrd to disk */
+        if (cbm_files_match(kernel->initrd_file, initrd_target)) {
+                return true;
+        }
+
```

Comment:
May just be me, but I think it'd be nice if the style matched for kernel and initrd tests (either nest checks or split up checks for both).